modules/microshift-restart-ovnkube-master.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-restart-ovnkube-master_{context}"]
+= Restarting the ovnkube-master pod
+
+The following procedure restarts the ovn-kube master pod, which also restarts the node.
```

Comment:
"which also restarts the node" is not accurate. The steps below only restart the ovn-master pod. A node refers to a virtual or physical machine. Whereas a pod is a set of running containers. Containers run on a node.

Reply:
Ah OK, I thought the ovnkube-master restart was making the node restart so that it found the changed value. (Following the earlier comments on the Jira by @zshi-redhat about restarting the node.) So we need to do two things: 1) Restart ovnkube to adopt the value change, 2) then reboot the host to apply the new packet size to all the pods?